packages/nova-react/src/scheduling/nova-scheduling-provider.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import type { NovaScheduling } from "@nova/types";
+import invariant from "invariant";
+
+// Initializing default with null to make sure providers are correctly placed in the tree
+const NovaSchedulingContext = React.createContext<NovaScheduling | null>(null);
+
+interface NovaSchedulingProviderProps {
+  scheduling: NovaScheduling;
+}
+
+export const NovaSchedulingProvider: React.FunctionComponent<
+  NovaSchedulingProviderProps
+> = ({ children, scheduling }) => {
+  return (
+    <NovaSchedulingContext.Provider value={scheduling}>
```

Comment:
Is it possible to add a default value for scheduling to support hosts which simple needs? We don't have any examples in this PR of how we expect a host to hook this, or a default implementation... :)